torch/onnx/_internal/exporter/_torchlib/ops/hop.py

```diff
@@ -92,3 +92,65 @@ def higher_order_cond(
             (), else_node.outputs, nodes=[else_node], name=false_func.name
         ),
     )
+
+
+@onnx_impl(torch.ops.higher_order.scan, no_compile=True)
+def higher_order_scan(
```

Comment:
hmmm I still don't see the torch op schema link

Reply:
I wasn't able to find a schema, just an implementation 🤔

Reply:
I think https://github.com/pytorch/pytorch/blob/66ac724b56e6c37a534f3e066423ef2f41d7477f/torch/_higher_order_ops/scan.py#L109 is good

Reply:
Done